### PR TITLE
[*]Selenium_Lesson2

### PR DESCRIPTION
**Pull request by krukelena**
Add selenium test

### CODE REVIEW

File: NunitTest_Selenium/NunitTest_Selenium/Checkboxes.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+        [SetUp]
+        public void SetUp()
+        {
+            ChromeDriver = new ChromeDriver();
+            ChromeDriver.Manage().Window.Maximize();
+            ChromeDriver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
+            ChromeDriver.Navigate().GoToUrl("http://the-internet.herokuapp.com/checkboxes");
+        }
+        [Test]
```
Comment (VladimirVolkovVD) [inline NunitTest_Selenium/NunitTest_Selenium/Checkboxes.cs:26]: add empty line

File: NunitTest_Selenium/NunitTest_Selenium/Checkboxes.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+            var secondCheckboxAttribute = secondCheckbox.GetAttribute("checked");
+            Assert.IsNotNull(secondCheckboxAttribute);
+
+            secondCheckbox.Click();
+
+            secondCheckboxAttribute = secondCheckbox.GetAttribute("checked");
+            Assert.IsNull(secondCheckboxAttribute);
+
+        }
```
Comment (VladimirVolkovVD) [inline NunitTest_Selenium/NunitTest_Selenium/Checkboxes.cs:50]: add empty line

File: NunitTest_Selenium/NunitTest_Selenium/Dropdown.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        WebDriver ChromeDriver { get; set; }
+
+        [SetUp]
+        public void SetUp()
+        {
+            ChromeDriver = new ChromeDriver();
+            ChromeDriver.Manage().Window.Maximize();
+            ChromeDriver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
+            ChromeDriver.Navigate().GoToUrl("http://the-internet.herokuapp.com/dropdown");
+        }
```
Comment (VladimirVolkovVD) [inline NunitTest_Selenium/NunitTest_Selenium/Dropdown.cs:28]: add empty line

File: NunitTest_Selenium/NunitTest_Selenium/NotificationMessages.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+     
+            Assert.That(flashElement.Text, Is.EqualTo("Action successful\r\n×"));
+            
+            
+
+        }
+        public void TearDown()
+        {
+            //ChromeDriver.Quit();
+        }
```
Comment (VladimirVolkovVD) [inline NunitTest_Selenium/NunitTest_Selenium/NotificationMessages.cs:43]: do we need to close browser?

File: NunitTest_Selenium/NunitTest_Selenium/NotificationMessages.cs [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        [SetUp]
+        public void SetUp()
+        {
+            ChromeDriver = new ChromeDriver();
+            ChromeDriver.Manage().Window.Maximize();
+            ChromeDriver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
+            ChromeDriver.Navigate().GoToUrl("https://the-internet.herokuapp.com/notification_message_rendered");
+
+        }
+        [Test]
```
Comment (VladimirVolkovVD) [inline NunitTest_Selenium/NunitTest_Selenium/NotificationMessages.cs:28]: add empty line

File: NunitTest_Selenium/NunitTest_Selenium/NotificationMessages.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        {
+           // var expectedMessage = $"Action successful\n+×";
+            var linkButton = ChromeDriver.FindElement(By.CssSelector(".example a"));
+            linkButton.Click();
+            var flashElement = ChromeDriver.FindElement(By.Id("flash"));
+     
+            Assert.That(flashElement.Text, Is.EqualTo("Action successful\r\n×"));
+            
+            
+
```
Comment (VladimirVolkovVD) [inline NunitTest_Selenium/NunitTest_Selenium/NotificationMessages.cs]: extra lines

File: NunitTest_Selenium/NunitTest_Selenium/SortableDataTables.cs [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+        [SetUp]
+        public void SetUp()
+        {
+            ChromeDriver = new ChromeDriver();
+            ChromeDriver.Manage().Window.Maximize();
+            ChromeDriver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
+            ChromeDriver.Navigate().GoToUrl("https://the-internet.herokuapp.com/tables");
+
+        }
+        [Test]
```
Comment (VladimirVolkovVD) [inline NunitTest_Selenium/NunitTest_Selenium/SortableDataTables.cs:26]: add empty line

File: NunitTest_Selenium/NunitTest_Selenium/Typos.cs [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+        [SetUp]
+        public void SetUp()
+        {
+            ChromeDriver = new ChromeDriver();
+            ChromeDriver.Manage().Window.Maximize();
+            ChromeDriver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
+            ChromeDriver.Navigate().GoToUrl("http://the-internet.herokuapp.com/typos");
+
+        }
+        [Test]
```
Comment (VladimirVolkovVD) [inline NunitTest_Selenium/NunitTest_Selenium/Typos.cs:27]: add empty line

**Review by VladimirVolkovD — CHANGES_REQUESTED** [review]

**Pull request merged**